src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Figureable {
```

Comment:
all you need for Figure class if a field, constructor and getter. We don't need to explicitly add here methods of the interfaces, it's enough to just `implement` them. Our figures will override them